### ISSUE TITLE
Error installing ReText on Windows: -single-version-externally-managed not recognized

### ISSUE THREAD

**Issue by taa1** [retext-project/retext#324]
Windows 10 Home
Python 3.6.2, 32bit

Attempting install with:
`python -m pip install ReText`

Log is at https://pastebin.com/iEHcjKtJ

I've tried installing as administrator (you'll see "Requirement already satisfied") and get the same result. 

Brand new install of Python.

I realize the Windows version of ReText is not officially supported. I'm hoping the error is not platform specific but I don't know for sure.

**Comment by mitya57** [comment]
This is duplicate of #295. It will be fixed in the next release.

As a workaround you can install the version from Git with `setup.py install`.

**Comment by taa1** [comment]
Thanks, mitya57, that took care of it. The toolbar icons were missing but I found them (step 4 below). In case anyone else has this issue on Windows before the next release of ReText, I did:

1. Download https://github.com/retext-project/retext/archive/master.zip to a temporary folder.
2. Unzip it.
3. Changed to the temporary folder and did: **python setup.py install**
4. Download the icon pack https://github.com/retext-project/retext/archive/icons.tar.gz. to a temporary folder.
5. Extract the icon pack (I used [7zip](http://www.7-zip.org/download.html)).
6. Copy the icons. Assuming you're in the temporary folder:
    copy retext-icons\ C:\Python36-32\share\retext\icons\
